wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+
+**Actions:**
+
+* if the ISIN is a "MissingValue" because it is not available in the BBG database, there is nothing we can do (??)
+* if the ISIN seems to have an odd format, check (potentially with the project partner) if the identifier is really aan ISIN, or some other identifier such as a FIGI. If so, update the identifier.
+* the invalid ISIN might also just be a typo, verify this for cases that look close to valid.
+* These actions are taken on the portfolio input level, so no need to involve the data team
+
+```{r}
+ISIN_missing_invalid <- result_data_qa %>% 
+  filter((flag == "Invalid or missing ISIN" | isin == "MissingValue") &
+           investor_name != "Meta Investor") %>% 
+  select(investor_name, portfolio_name, isin, company_name,  holding_id, #market_value, currency,exchange_rate,
+         value_usd, asset_type, financial_sector, fund_isin, flag) %>% 
+  group_by(fund_isin) %>% 
+  mutate(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(
+    share_missing_value = round(
+      sum_value_usd/(investors_overview %>% filter(investor_name=="Meta Investor") %>% pull(sum_value_usd)), 7
+      )
+    ) %>% 
+  arrange(desc(share_missing_value), fund_isin)
+datatable(ISIN_missing_invalid)
+
+```
+
+## Missing ALD
+
+### overview Equity holdings w/o ALD
+Overview of Holdings in PACTA sectors for which no ALD could be matched.
+
+1. Overview on sector level, including count and share of overall equity value affected
+2. Overview on sector-company level, including overall equity value affected
+
+**Actions:**
+
+* if there are companies present, for which we should have corresponding asset-level data, check in DataStore if the source of the issue can be located. Possibly involve Taylor to fix such mapping
+* other steps???
+
+```{r}
+no_ALD_EQ <- result_data_qa %>% 
+  filter(asset_type == "Equity" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  group_by(financial_sector) %>% 
+  summarise(total_no_ALD_EQ = n(), sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Equity") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector)
+datatable(no_ALD_EQ)
+
+no_ALD_EQ_detail <- result_data_qa %>% 
+  filter(asset_type == "Equity" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  select(financial_sector, isin, company_name, market_value, value_usd) %>% 
+  group_by(financial_sector, isin, company_name) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Equity") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_EQ_detail)
```

Comment:
This is great! I hope it is actually used, but this generally makes lots of sense!